### PR TITLE
feat: add clone method to IPoint type definition

### PR DESCRIPTION
**Pull request by MMasey**
##### Description of change
I noticed that the IPoint interface was missing the clone method so this PR adds it in.

### CODE REVIEW

File: packages/math/src/IPoint.ts [hunk 1]
```diff
@@ -35,6 +36,13 @@ export interface IPoint
  * @param {number} [y=x] - position of the point on the y axis
  */
 
+/**
+ * Creates a clone of this point
+ * @method clone
+ * @memberof PIXI.IPoint#
+ * @return {PIXI.Point} a copy of the point
```
Comment (bigtimebuddy) [inline packages/math/src/IPoint.ts]: Should the return value be IPoint?

Comment (MMasey) [reply packages/math/src/IPoint.ts]: Hi @bigtimebuddy, you're correct. I've just pushed up a fix 👍

**Comment by bigtimebuddy** [comment]
@ivanpopelyshev what are you proposing?

**Comment by ivanpopelyshev** [comment]
`ObservablePoint.clone()` should return `Point` and not `ObservablePoint`

**Comment by bigtimebuddy** [comment]
Am closing this PR because `clone` method signatures are not the same between Point and ObservablePoint. Having ObservablePoint clone to Point is a breaking change. Would prefer to leave this as-is and have users add [type assertions](https://www.typescriptlang.org/docs/handbook/basic-types.html#type-assertions) for their particular IPoint objects when cloning.

**Pull request closed without merging**